src/Dfc.ProviderPortal.ChangeFeedListener/Settings/FaocSearchServiceSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Dfc.ProviderPortal.ChangeFeedListener.Interfaces;
+
+namespace Dfc.ProviderPortal.ChangeFeedListener.Settings
+{
+    public class FaocSearchServiceSettings : IFaocSearchServiceSettings
+    {
+        public string SearchService { get; set; }
```

Comment:
Not sure we need most of this - key is probably sufficient

Reply:
removed most of them now, kept the 4 being used.